main.go

```diff
@@ -609,19 +664,35 @@ func ParseAllThreeLists() *cobra.Command {
 						maxPodConcurrency = retI3[iii]
 					}
 				}
-
-				fmt.Fprintf(os.Stdout, "PipelineRun %s\t\t took %v seconds with pr concurrency %d with taskruns %v seconds delta %v percent %f taskrun max concurrency %d pods %v seconds delta %v percent %f pod max concurrency %d\n",
-					prkey,
-					prDuration,
-					prConcurency,
-					totalTRDuration,
-					prDuration-totalTRDuration,
-					totalTRDuration/prDuration,
-					maxTRConcurrency,
-					totalPodDuration,
-					prDuration-totalPodDuration,
-					totalPodDuration/prDuration,
-					maxPodConcurrency)
+				switch outputType {
+				case OutputTypeCsv:
+
+					fmt.Fprintf(os.Stdout, "%s;%v;%d;%v;%v;%f;%d;%v;%v;%f;%d\n",
```

Comment:
so yeah this usage is why you need the variadic function on the utility method I'm asking for